### PR TITLE
Added know issue section.

### PR DESCRIPTION
**Pull request by seriva**

### CODE REVIEW

File: CHANGELOG-0.6.md [hunk 1]
```diff
@@ -61,3 +61,11 @@
 - [#1063](https://github.com/epiphany-platform/epiphany/issues/1063) - Issues with single_machine install
 - [#1108](https://github.com/epiphany-platform/epiphany/issues/1108) - [Azure RedHat] Create epirepo: package httpd-2.4.6-93 requires httpd-tools = 2.4.6-93 but latest available is 2.4.6-90
 - [#1110](https://github.com/epiphany-platform/epiphany/issues/1110) - Install fixed version of httpd when latest fails (RHEL)
+
+### Known Issues
+
+- [#1068](https://github.com/epiphany-platform/epiphany/issues/1068) - K8s HA installation - failing in some cases on task "Get token from master
```
Comment (plirglo) [inline CHANGELOG-0.6.md:67]: " at the end of line

File: CHANGELOG-0.6.md [hunk 2]
```diff
@@ -61,3 +61,11 @@
...
+- [#1075](https://github.com/epiphany-platform/epiphany/issues/1075) - K8s HA installation - timed out on task "Join master to ControlPlane
```
Comment (plirglo) [inline CHANGELOG-0.6.md:68]: " at eol

File: CHANGELOG-0.6.md (hunk 1 above)
Comment (seriva) [reply CHANGELOG-0.6.md]: fixed

File: CHANGELOG-0.6.md (hunk 2 above)
Comment (seriva) [reply CHANGELOG-0.6.md]: fixed

**Pull request merged**